.github/workflows/pull.yml

```diff
@@ -302,26 +302,6 @@ jobs:
       docker-image-name: pytorch-linux-focal-py3.7-gcc7
       build-generates-artifacts: false
 
-  linux-bionic-cuda11_6-py3_10-gcc7-deploy-build:
```

Comment:
do you think internal tests are enough or do we want to add a sanity check to build multipy from torch to help avoid regressions?

Reply:
Personally I would, but after talking to @seemethere, didn't opt to as it would make the pytorch devX worse by adding more test time. Though if the test is lightweight, it may be reasonably to add a sanity check for this reason.